GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView.swift

```diff
@@ -0,0 +1,172 @@
+import UIKit
+
+extension CallVisualizer {
+    internal final class VisitorCodeView: BaseView {
```

Comment:
this class is internal by default, so let's drop this redundant `internal` keyword.

Reply:
fixed [676c74c](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/676c74ceb781899d4ca4fa8a8ab684c8a1837308)